### ISSUE TITLE
Grammartical typo in footer section of the themeConfig page

### ISSUE THREAD

**Issue by Evavic44** [vuejs/vitepress#1606]
In the `themeConfig` section, under [footer](https://vitepress.vuejs.org/config/theme-configs#footer), there seems to be a grammatical mistake. 

### Ergo

Footer configuration. You can add a message and copyright. The footer will `displayed` only when the page doesn't `contain sidebar` due to `design reason`.

## Suggested change
Footer Configuration. You can add a message or copyright text on the footer, however, it will only be displayed when the page doesn't contain a sidebar. This is due to design concerns.

I just paraphrased the sentence a little bit, but it can be simplified even further. Also, if approved, I can make a quick PR for this.

**Comment by brc-dd** [comment]
Yeah feel free to create a PR. There isn't any need to open an issue at first for most of the docs related stuff.

**Comment by Evavic44** [comment]
> Yeah feel free to create a PR. There isn't any need to open an issue at first for most of the docs related stuff.

Noted, @brc-dd making a PR now. 🙂
